RELEASE_PROCESS.md

```diff
@@ -39,15 +41,34 @@ Every commit to chef-server master is tested against a full pedant
 run. However, upgrade testing must still be done in advance of the
 release:
 
-- [ ] Run the Chef Delivery-based automated upgrade testing
```

Comment:
Step 1 should probably be letting #cft-announce know that you're going to release some stuff... or something like that...

Maybe I should go collect information on whether it's a minor bump or a patch bump and a list of changes?  But where would I ask?  I guess it's "ask #chef-server/spool for important changes to be highlighted and then let #cft-announce know that you're up to some mischief"